python_tests/test_python_frontend.py

```diff
@@ -62,19 +62,16 @@ def inner(*args, **kwargs):
         # Run test to populate FusionCache
         test_fn(*args, **kwargs)
 
-        # Delete previous file
-        if os.path.isfile("foo.bin"):
-            os.remove("foo.bin")
+        with tempfile.NamedTemporaryFile() as tmp:
+            # Serialize FusionCache
+            fc = FusionCache.get()
+            fc.serialize(tmp.name)
 
-        # Serialize FusionCache
-        fc = FusionCache.get()
-        fc.serialize("foo.bin")
+            FusionCache.reset()
 
-        FusionCache.reset()
-
-        # Get new FusionCache because the previous one was destroyed by the reset call.
-        fc = FusionCache.get()
-        fc.deserialize("foo.bin")
+            # Get new FusionCache because the previous one was destroyed by the reset call.
+            fc = FusionCache.get()
+            fc.deserialize(tmp.name)
```

Comment:
Just cleaning up the `foo.bin` stuff while I'm here. Reverting that commit has no effect on the error I'm seeing.